app/controllers/user_controllers.js

```diff
@@ -91,19 +29,18 @@ exports.newUser = (req, res, next) => {
       }
     : {};
 
-  let valMsg = validateUser(newUser);
-  if (valMsg.length > 0) {
-    valMsg = valMsg.slice(0, -1);
-    next(errors.defaultError(valMsg));
+  const valMsgs = validation.validateUser(newUser);
+  if (valMsgs.length > 0) {
+    next(errors.defaultError(valMsgs));
   } else {
     bcrypt
       .hash(newUser.password, saltRounds)
       .then(hash => {
         newUser.password = hash;
 
         User.createNewUser(newUser)
-          .then(u => {
-            res.send(`Successfully logged in. Welcome, ${u.firstName} ${u.lastName}!`);
+          .then(user => {
+            res.send(`Successfully logged in. Welcome, ${user.firstName} ${user.lastName}!`);
```

Comment:
This should be something like `res.status(201).end()`, message is not required for functionality.